docs/source/development/ipython_directive.rst

```diff
@@ -0,0 +1,435 @@
+
+.. _ipython_directive:
+
+========================
+IPython Sphinx Directive
+========================
+
+.. note::
+
+   This has been salvadged from history, so information may be approximate or
```

Comment:
I'd just say something like "The IPython Sphinx Directive is in 'beta' and currently under active development. Improvements to the code or documentation are welcome!"